src/components/Playground.tsx

```diff
@@ -232,6 +240,21 @@ class Playground extends React.Component<IPlaygroundProps, PlaygroundState> {
       />
     );
 
+    const playgroundIntroductionTab: SideContentTab = {
+      label: 'Introduction',
+      iconName: IconNames.COMPASS,
+      body: (
+        <Markdown
+          content={
+            INTRODUCTION +
+            (this.props.sourceVariant === 'concurrent' ? CONCURRENT_SOURCE_INTRODUCTION : '')
```

Comment:
Nice to see a scheme how we can evolve these intro sections and make them specific to the particular Source language.